docs/book/platform-guide/set-up-your-mlops-platform/deploy-zenml/deploy-with-custom-image.md

```diff
@@ -0,0 +1,126 @@
+---
+description: Deploying ZenML with custom Docker images.
+---
+
+In most cases, deploying ZenML with the default `zenmlhub/zenml-server` Docker 
+image should work just fine. However, there are some scenarios when you 
+might need to deploy ZenML with a custom Docker image:
+- You have implemented a custom artifact store for which you want to enable
+[artifact visualizations](../../../user-guide/advanced-guide/visualize-artifacts.md)
+or [step logs TODO LINK](../../../user-guide/advanced-guide/visualize-artifacts.md)
```

Comment:
LINK

Reply:
Yep this needs to wait till the step logs page exists (which @bcdurak will write this sprint)

Reply:
Actually I just checked and we don't have a ticket for that, we only have a ticket to document how the logs can be switched off which will not become a separate section I assume.

I linked it to best fitting section we have for now.